content/docs/nextjs/github-public-repo.md

```diff
@@ -138,34 +178,37 @@ export const EditLink = ({ editMode }: EditLinkProps) => {
 
 > _Note:_ For brevity, the example above configures many steps in a single file, but **a few components can be configured in different places**. For example you could put the `EditLink` in a Layout component, or setup the Github Provider only on certain pages.
 
-Now that **\_app.tsx** is ready. Let's setup up the backend API.
+If you restart the dev server, you should see a **button in the top left-hand corner** that says, "Edit This Site". If you click it, you'll be prompted to authenticate with GitHub, but we still have some steps to get this working. Let's setup up the backend API.
 
-## API functions
+## Adding API functions
 
 We will need a few API functions to handle GitHub authentication and [Preview Mode](https://nextjs.org/docs/advanced-features/preview-mode). With Next.js, any functions in the `pages/api` directory are are mapped to `/api/*` endpoints.
 
-We've created a few [scripts](https://github.com/dwalkr/hygen-next-tinacms-github) to **help generate the required files**. From the console, run:
+We've created a few [scripts](https://github.com/dwalkr/hygen-next-tinacms-github) to **help generate the required files**. From the terminal, run:
```

Comment:
Maybe it's because I'm a JS dummy, but I'm relatively uncertain as to what this does.

Reply:
Well, I guess if I keep reading I end up understanding.

Reply:
That's a good point, could use more explaining there

Reply:
I added a _bit_ of explanation by linking to the project and noting that Hygen is a code generator. Let me know if that helps contextualize this step.